packages/tasks/src/model-libraries.ts

```diff
@@ -715,6 +715,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		countDownloads: `path:"model_vae_fp16.pt"`,
 		snippets: snippets.threedtopia_xl,
 	},
+	cotracker: {
+		prettyLabel: "cotracker",
+		repoName: "cotracker",
```

Comment:
```suggestion
		prettyLabel: "CoTracker",
		repoName: "CoTracker",
```
for what I can read on https://github.com/facebookresearch/co-tracker and https://co-tracker.github.io/, `"CoTracker"` seems to be the preferred casing.

Reply:
Indeed, we usually use CoTracker. 

Curious what would it affect in this case? Right now our HF model (https://huggingface.co/facebook/cotracker/tree/main) tagged with `cotracker`, if we commit this change, shall we change tag as well in our existing model and use this tag in the future releases?

Reply:
@Wauplin ping :)

Reply:
Oops sorry for the delay @lvoursl  :hugs:

Your model been tracked as `cotracker` is good. In the PR changes you've made:
- `cotracker` (the key of the mapping you've added) is the tag that the model should have. Example [here](https://huggingface.co/google/gemma-2b-keras/blob/main/README.md?code=true#L2) with `keras-nlp`.
- `prettyName` is how it will be displayed for users. See example with `KerasNLP`:
![image](https://github.com/user-attachments/assets/a86fbe87-71cd-4f46-a88c-2da46caea4ec)
- `repoName` (usually the same as `prettyName`) is how the library is called for the ["use this model" snippet](https://huggingface.co/google/gemma-2b-keras/blob/main/README.md?code=true&library=keras-nlp#L2):
![image](https://github.com/user-attachments/assets/cf426f7e-5b25-47e6-aa2b-2278166a10f2)

Reply:
Thanks for the clarification @Wauplin, appreciate it! I've committed the changes that you proposed, let me know if there anything else has to be done to merge it.